website/content/terminal/usage/basics/index.md

```diff
@@ -1,114 +1,292 @@
+
+### Importing and Exporting Data
+
+Most functions provide a method for exporting the raw data as a CSV, JSON, or XLSX file.  Exported data and user-supplied files to import are saved to the [OpenBBUserData folder](https://docs.openbb.co/terminal/usage/guides/data).  The folder is located at the root of the operating system's User Account folder.  Follow the link for a detailed description.
+
+## Charts
+
+The OpenBB charting library has provides interactive, customizable, charts.  Here's an example of weekly candles for AAPL.
+
+```console
+/stocks/load AAPL -w/candle
+```
+
+![Apple Weekly Chart](https://user-images.githubusercontent.com/85772166/233247951-e011fe2c-23a6-4518-bd17-3f43a9c2011a.png)
+
+### Toolbar
+
+The toolbar is located at the bottom of the window, and provides methods for:
+
+- Panning and zooming.
+- Modifying the title and axis labels.
+- Adjusting the hover read out.
+- Toggling light/dark mode.
+- Annotating and drawing.
+- Exporting raw data.
+- Saving the chart as an image.
+- Adding supplementary external data as an overlay.
+
+The label for each tool is displayed by holding the mouse over it.
+
+![Chart Tools](https://user-images.githubusercontent.com/85772166/233247997-55c03cbd-9ca9-4f5e-b3fb-3e5a9c63b6eb.png)
+
+Toggle the toolbar's visibility via the keyboard with, `ctrl + h`.
+
+### Text Tools
+
+Annotate a chart by clicking on the `Add Text` button, or with the keyboard, `ctrl + t`.
+
+![Annotate Charts](https://user-images.githubusercontent.com/85772166/233248056-d459d7a0-ba2d-4533-896a-79406ded859e.png)
+
+Enter some text, make any adjustments to the options, then `submit`.  Place the crosshairs over the desired data point and click to place the text.
+
+![Place Text](https://user-images.githubusercontent.com/85772166/233248107-758bc0b2-ac92-451e-aabb-16eb111c7f88.png)
+
+After placement, the text can be updated or deleted by clicking on it again.
+
+![Delete Annotation](https://user-images.githubusercontent.com/85772166/233248228-6226ab12-46dd-443b-a7c4-145fc781e71d.png)
+
+The title of the chart is edited by clicking the button, `Change Titles`, near the middle center of the toolbar, immediately to the right of the `Add Text` button.
+
+### Draw Tools
+
+The fourth group of icons on the toolbar are for drawing lines and shapes.
+
+- Edit the colors.
+- Draw a straight line.
+- Draw a freeform line.
+- Draw a circle.
+- Draw a rectangle.
+- Erase a shape.
+
+To draw on the chart, select one of the four drawing buttons and drag the mouse over the desired area.  Click on any existing shape to modify it by dragging with the mouse and editing the color, or remove it by clicking the toolbar button, `Erase Active Shape`.  The edit colors button will pop up as a floating icon, and clicking on that will display the color palette.
+
+![Edit Colors](https://user-images.githubusercontent.com/85772166/233248349-d7cd2e2e-7e22-4efb-acdb-5f7a85bf2932.png)
```

Comment:
I know this is the default stuff, but this is very hard to read on the chart.  NOt sure if you can change either to a light theme chart or a lighter font color

Reply:
Updated charts, much easier it to read now.